lib/l10n/intl_messages.arb

```diff
@@ -3101,5 +3101,65 @@
     "type": "text",
     "placeholders_order": [],
     "placeholders": {}
+  },
+  "camera": "Camera",
+  "@camera": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "audio_recording": "Audio recording",
+  "@audio_recording": {
+    "type": "text",
+    "placeholders_order": [],
+    "placeholders": {}
+  },
+  "give_access_in_settings": "Please give linshare access in the settings to continue",
```

Comment:
`LinShare`, `Settings`

Reply:
And we should more detail on what `permission` and why we need it? This is a chance for clearer information to user.

Reply:
Updated

![image](https://github.com/user-attachments/assets/78c5ff25-729c-4345-9d17-2f0cbc37c0a2)